compiler/resolution/resolveFunction.cpp

```diff
@@ -620,12 +620,6 @@ static void resolveAlsoConversions(FnSymbol* fn, CallExpr* forCall) {
     // arg 1 is the method token
     toType = fn->getFormal(2)->getValType();
     fromType = fn->getFormal(3)->getValType();
-  } else if (fn->name == astr_cast) {
```

Comment:
Arguably this should be considering `fn->name == astrScolon`. You could look at the `astrSassign` case above for how to ignore the type-function arguments. Then, the formals are `fromValue` and `toType`.

(I think the code will work without this - so this change is optional, but arguably doing it here makes the code more straightforward. Or at least that's what I was thinking when I wrote it...)

Reply:
All tests seem to pass without it, so I'm going to remove it on the basis of "removing code = good".